rust/src/ffi/steppable_evmc_vm.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(clippy::undocumented_unsafe_blocks)]
```

Comment:
we do have document lines inside of unsafe code in this file. please add the missing ones and remove the exception.